content/influxdb/clustered/query-data/troubleshoot-and-optimize/report-query-performance-issues.md

```diff
@@ -310,3 +323,71 @@ curl --get "https://{{< influxdb/host >}}/query" \
 {{< /code-tabs-wrapper >}}
 
 {{% /code-placeholders %}}
+
+### Gather system information
+
+{{% warn %}}
+#### May impact cluster performance
+
+Querying InfluxDB v3 system tables may impact write and query
+performance of your {{< product-name omit=" Clustered" >}} cluster.
+Use filters to [optimize queries to reduce impact to your cluster](/influxdb/clustered/admin/query-system-data/#optimize-queries-to-reduce-impact-to-your-cluster).
+
+<!--------------- UPDATE THE DATE BELOW AS EXAMPLES ARE UPDATED --------------->
+
+#### System tables are subject to change
+
+System tables are not part of InfluxDB's stable API and may change with new releases.
+The provided schema information and query examples are valid as of **September 20, 2024**.
+If you detect a schema change or a non-functioning query example, please
+[submit an issue](https://github.com/influxdata/docs-v2/issues/new/choose).
+
+<!--------------- UPDATE THE DATE ABOVE AS EXAMPLES ARE UPDATED --------------->
+{{% /warn %}}
+
+If queries are slow for a specific table, run the following system queries to collect information for troubleshooting.
+
+- [Collect table information](#collect-table-information)
+- [Collect compaction information for the table](#collect-compaction-information-for-the-table)
+- [Collect partition information for multiple tables](#collect-partition-information-for-multiple-tables)
+
+In your queries, replace the following:
+
+- {{% code-placeholder-key %}}`TABLE_NAME`{{% /code-placeholder-key %}}: the
+  table to retrieve information about 
+
+#### Collect table information
+
+{{% code-placeholders "TABLE_NAME" %}}
+```sql
+SELECT *
+FROM system.tables
+WHERE table_name = 'TABLE_NAME';
+```
+{{% /code-placeholders%}}
+
+#### Collect compaction information for the table
+
+{{% code-placeholders "TABLE_NAME" %}}
+```sql
+SELECT *
+FROM system.compactor
+WHERE table_name = 'TABLE_NAME';
+```
+{{% /code-placeholders%}}
+
+#### Collect partition information for multiple tables
+
+If the same queries are slow on more than 1 table, also run the following query to collect the size and
+number of partitions for all tables:
+
+{{% code-placeholders "TABLE_NAME" %}}
+```sql
+SELECT table_name,
+  COUNT(*) as partition_count,
+  MAX(last_new_file_created_at) as last_new_file_created_at,
+  SUM(total_size_mb) as total_size_mb
+FROM system.partitions
+GROUP BY table_name;
+```
+{{% /code-placeholders%}}
```

Comment:
Probably add a link to the section "Optimize queries to reduce impact to your cluster" can be helpful: 

https://docs.influxdata.com/influxdb/clustered/admin/query-system-data/#optimize-queries-to-reduce-impact-to-your-cluster